coral-dbt/src/main/resources/tests/test_incremental.py

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright 2023 LinkedIn Corporation. All rights reserved.
+# Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+#
+import subprocess
+import unittest
+from jinja2 import Template
+
+SEEDS_DIR = 'seeds/'
+URL = 'http://localhost:8080/api/incremental/rewrite'
+
+class TestUtils:
+    @staticmethod
+    def construct_generate_incremental_code_test(name):
+        output_table = name + '_output'
+        expected_file = open(SEEDS_DIR + 'test_' + name + '_expected.txt', 'r')
+        expected_incremental_spark_scala = expected_file.read()
+        expected_file.close()
+
+        return({
+            'output_table': output_table,
+            'expected_incremental_spark_scala': expected_incremental_spark_scala,
+        })
+
+    @staticmethod
+    def test_generate_incremental_code(coral_response, table_names, output_table):
+        with open('generate_incremental_code.j2') as f:
+            template = Template(f.read())
+
+        context = {'coral_response': coral_response, 'table_names': table_names, 'output_table': output_table}
+        output = template.render(context)
+
+        return output
+
+class TestIncremental(unittest.TestCase):
```

Comment:
These tests are running as a Gradle task as part of the overall build process. Can you also add steps for how to run the tests separately and from which directory they can be run?